regen/warnings.pl

```diff
@@ -962,6 +973,41 @@ =head2 Default Warnings and Optional Warnings
 Note that neither the B<-w> flag or the C<$^W> can be used to
 disable/enable default warnings.  They are still mandatory in this case.
 
+=head2 "Negative warnings"
+
+As a convenience, you can as of Perl 5.34 pass arguments to the
```

Comment:
Grammatical nit: I think “as of Perl 5.34” should have commas around it.

(Same applies in warnings.pm.)

Reply:
that would, I suggest, lead, unintentionally, to an unfortunate case of [shatner comma](https://latimesblogs.latimes.com/jacketcopy/2011/06/goodbye-oxford-comma-hello-shatner-comma.html) - previous text seemed fine, but if you really want to break it up then why not `(as of Perl 5.34)`.

Reply:
> 3. Use a pair of commas in the middle of a sentence to set off clauses, phrases, and words that are not essential to the meaning of the sentence.

https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html

Reply:
I do realise that in the time I've taken to type this I could have made the changes and committed them. I haven't, because the aim of review is to *improve* a PR, not to impose some individual's preferences. I'm sure that mechanically following those rules is good for non-native speakers wishing to produce clear if somewhat stilted prose, but that's all that mechanically following rules would be good for.

As a native English speaker for the past 40+ years I say with utter conviction that extra commas are unnecessary there. A splattering of extra commas wouldn't change the meaning in any way and, just like @tom-binary's example, can only, in this case, harm comprehensibility, just a little bit, by breaking up the flow.

Reply:
@DrHyde The reason why I posted the link in my last response was to illustrate that this is not an individual preference. It’s just standard English.

Reply:
As a native English speaker, I agree that commas unnecessarily break the flow but parentheses improve it greatly.

Reply:
As yet another native English speaker :), I agree that parens would be an improvement, probably better than the (otherwise requisite) commas.